flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/DataSkewStreamNetworkThroughputBenchmark.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.runtime.io.benchmark;
+
+import org.apache.flink.core.io.IOReadableWritable;
+import org.apache.flink.runtime.io.network.api.writer.ChannelSelector;
+import org.apache.flink.runtime.io.network.api.writer.RecordWriterBuilder;
+import org.apache.flink.runtime.io.network.api.writer.ResultPartitionWriter;
+
+/**
+ * Network throughput benchmarks for data skew scenario executed by the external
+ * <a href="https://github.com/dataArtisans/flink-benchmarks">flink-benchmarks</a> project.
+ */
+public class DataSkewStreamNetworkThroughputBenchmark extends StreamNetworkThroughputBenchmark {
+
+	@Override
+	protected LongRecordWriterThread createWriterThread(
```

Comment:
nit: I think it would be slightly cleaner and with a bit less code duplication to not introduce & override the `LongRecordWriterThread createWriterThread(...)` method, but maybe
```
@Override
RecordWriterBuilder setChannelSelector(RecordWriterBuilder recordWriterBuilder, boolean broadcastMode) {
  checkArgument(!broadcastMode, "Combining broadcasting with data skew doesn't make sense");
  return recordWriterBuilder.setChannelSelector(new DataSkewChannelSelector());
}
```
?

Reply:
Fixed.